x-pack/platform/plugins/shared/alerting/server/routes/maintenance_window/apis/external/create/create_maintenance_window_route.ts

```diff
@@ -53,6 +53,7 @@ export const createMaintenanceWindowRoute = (
       options: {
         access: 'public',
         summary: 'Create a maintenance window.',
+        tags: ['oas-tag:alerting'],
```

Comment:
I am sure you are aware, but a note to not forget to resolve the conflicts with the public APIs PR where we changed the tags.